kimchi/src/verifier.rs

```diff
@@ -312,19 +314,24 @@ where
         //~ 1. Derive $v$ from $v'$ using the endomorphism (TODO: specify).
         let v = v_chal.to_field(endo_r);
 
+        println!("v: {}", v);
+
```

Comment:
Should we consider printing only in debug mode?

Reply:
Woops, caught me debugging :sweat_smile: